prompts/prompt_cot.md

```diff
@@ -1,9 +1,9 @@
 <|start_header_id|>user<|end_header_id|>
 
 Generate a SQL query to answer this question: `{user_question}`
-{instructions}{glossary}
+{instructions}
 DDL statements:
 {table_metadata_string}
 
-{cot_instructions}Generate a valid SQL query that answers the question `{user_question}`, and only references the tables and columns in the DDL statements.<|eot_id|><|start_header_id|>assistant<|end_header_id|>
+Generate a valid SQL query that answers the question `{user_question}`, and only references the tables and columns in the DDL statements.<|eot_id|><|start_header_id|>assistant<|end_header_id|>
```

Comment:
Shall we keep the `cot_instructions` field here? This is more for letting the model know that it should be generating the aliases followed by the SQL.
Currently in the data we use the instruction to let the model know when it should generate the table aliases, and when it should just directly generate the SQL (to facilitate the mixing of both types of data).

Reply:
Sounds good – making a fix now!